acquia/acquia_cms
# ACMS-1712: Enable drupal-check configurations with ignore specific errors.

**Motivation**

Fixes #[ACMS-1712](https://backlog.acquia.com/browse/ACMS-1712)

**Proposed changes**
Fix drupal-check issue reported in all acquia_cms modules

**Alternatives considered**
NA

**Testing steps**

- Get this PR code in your local
- Run command `php vendor/bin/drupal-check  modules/`

**Merge requirements**
